### ISSUE TITLE
iOS build error

### ISSUE THREAD

**Issue by njuxjy** [Tencent/wcdb#499]
### The language of WCDB

Swift

### The version of WCDB

latest commit in master branch, "8d71ce1c"

### The platform of WCDB

iOS, Xcode 10.2

### The installation of WCDB

Git clone

### What's the issue?

![491555579088_ pic_hd](https://user-images.githubusercontent.com/530251/56350717-01206b00-61fe-11e9-977f-7b0ae7edcbb7.jpg)

As shown above, I tried to build the WCDB.swift target but got error below: 'error: Build input file cannot be found: '/Users/hj/Desktop/projects/wcdb/sqlcipher/src/queue.c''

The sqlcipher submodule has been cloned but it looks like some files are missing from the  sqlcipher.xcodeproj sub-project.

**Comment by njuxjy** [comment]
After some investigation, it looks like incorrect commit HEAD has been specified for the sqlcipher git submodule. 
The current HEAD of sqlcipher is 7703ea7.  In this version of sqlcipher, there's no queue.c file within src folder, thus causing the build error. 
I checked the WCDBOptimizedSQLCipher.podspec file, and saw "src/queue.c" listed in the source_files section and also noticed this line: 

> sqlcipher.source       = { :git => "https://github.com/Tencent/sqlcipher.git", :commit => "7413f8517e305f4edf106dec0682515767f4fc4a" }

It looks like the commit that has queue.c (which is 7413f8517xxx here) is different from the current HEAD 7703ea7. 

After I changed the submodule HEAD to 7413f8517xxx, it worked like a charm.

Not sure if this is the root cause, but it fixed my issue. I will put a PR for this.
